app/services/intl.js

```diff
@@ -4,9 +4,9 @@
  */
 
 import Ember from 'ember';
-import Locale from 'ember-intl/models/locale';
-import createFormatCache from 'ember-intl/format-cache/memoizer';
 import { IntlRelativeFormat, IntlMessageFormat } from 'ember-intl/utils/data';
+import createFormatCache from 'ember-intl/format-cache/memoizer';
```

Comment:
I think there might have been a bug fix recently in the `intl-format-cache` package. Are you using that package? If not can we switch to using it? Otherwise you might want to make sure any bug fix that happened in that package is done here too.